### ISSUE TITLE
skin edema

### ISSUE THREAD

**Issue by sparkste** [obophenotype/human-phenotype-ontology#7005]
**Preferred term label:**
skin edema

**Synonyms**
integumentary edema

**Definition (free text, please give PubMed ID)**
thickening of the fetal skin due to edema

**Parent term (use hpo.jax.org/app)**
edema

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Noonan syndrome and other RASopathies
inborn errors of metabolism
lymphatic disorders
hematologic disorders
cardiac malformations
overgrowth disorders
ciliopathies
other single gene disorders leading to non-immune hydrops

**Your nano-attribution (ORCID)**
0000-0002-8593-2186

**Comment by pnrobinson** [comment]
@sparkste 
Is it correct to say that this term is different from the existing terms because there is no "dependent" edema for fetuses and the distribution of edema in the skin is different? What about ``Fetal skin edema`` as the primary label?
Also, hydrops has generalized skin edema -- is the current phenotypic feature also generalized skin edema or localized? Could we provide a more detailed definition?

**Comment by sparkste** [comment]
That's correct. The skin edema in a fetus is not "dependent." It can be generalized skin edema, or we can see that skin edema is present over the scalp, lower extremities, or only certain areas. Some cases with hydrops have generalized skin edema, others have skin edema only in certain areas of the body, and other cases with hydrops have no skin edema. Agree that "fetal skin edema" as the primary label makes sense.
Let me know if I can provide any further clarification. Thanks!



Teresa Sparks, MD, MAS

Assistant Professor

Maternal Fetal Medicine and Clinical Genetics

Department of Obstetrics, Gynecology, & Reproductive Sciences

University of California, San Francisco

________________________________

**Comment by patt1morgan** [comment]
@wtonyparks, @RoniZemet - Greetings! As you will see, there has already been some discussion around this new term request. It would be helpful if you could give us any suggestions you may have before we finalize this addition to the HPO. Thank you in advance for taking the time to review...

**Comment by pnrobinson** [comment]
This was added as Fetal skin edema. Thanks
